mlflow/keras/autolog.py

```diff
@@ -0,0 +1,261 @@
+# MLflow autologging support for Keras 3.
```

Comment:
This comment is useless. Let's remove it.

Reply:
Oh I made a mistake here, it should be a docstring instead of a comment. Basically according to Google python style guide, each module should have a top-level docstring.